eahub/templates/account/signup.html

```diff
@@ -33,11 +33,21 @@ <h1>Sign Up</h1>
             <div class="form-floating">
                 <input type="text"
                     class="form-control form-control-lg"
-                    name="{{ form.name.name }}"
+                    name="{{ form.first_name.name }}"
                     placeholder="Full Name"
-                    id="{{ form.name.id }}"
+                    id="{{ form.first_name.id }}"
                     required>
-                <label for="{{ form.name.id }}">Full Name</label>
+                <label for="{{ form.first_name.id }}">First Name</label>
+            </div>
+
+            <div class="form-floating">
+                <input type="text"
+                    class="form-control form-control-lg"
+                    name="{{ form.last_name.name }}"
+                    placeholder="Full Name"
+                    id="{{ form.last_name.id }}"
+                    required>
+                <label for="{{ form.last_name.id }}">Last Name</label>
             </div>
```

Comment:
Needs to be cleaned up: 
![image](https://user-images.githubusercontent.com/15908208/112749853-eeec0180-8fbc-11eb-8b25-bd2684232fd3.png)

Placeholders "FullName" are confusing

Reply:
a caching issue